utils/obsproc/Rads2Ioda.h

```diff
@@ -13,75 +13,49 @@
 
 #include "NetCDFToIodaConverter.h"
 
-#include "oops/util/missingValues.h"
-
 namespace gdasapp {
+
   class Rads2Ioda : public NetCDFToIodaConverter {
    public:
     explicit Rads2Ioda(const eckit::Configuration & fullConfig)
     : NetCDFToIodaConverter(fullConfig) {
-    oops::Log::info() << "Window begin: " << this->windowBegin_ << std::endl;
-    oops::Log::info() << "Window end: " << this->windowEnd_ << std::endl;
-    oops::Log::info() << "IODA output file name: " << this->outputFilename_ << std::endl;
     }
 
-    // Read netcdf file and populate group
-    void ReadFromNetCDF(ioda::Group & group) override {
-      std::string fileName = this->inputFilenames_[0];  // TODO(Guillaume): make it work for a list
+    // Read netcdf file and populate iodaVars
+    void ProviderToIodaVars(std::string fileName, gdasapp::IodaVars & iodaVars) override {
```

Comment:
could be `final` instead of `override`

Reply:
The `final` keyword goes in your Rads2Ioda subclass (not the base class). This will prevent anyone from creating a subclass of Rads2Ioda and overriding that method. Its probably not very likely that anyone would do this, but anyways...

Reply:
Ha, yes, I was clearly missing something.